gateway_code/open_nodes/tests/open_nodes_test.py

```diff
@@ -21,43 +21,79 @@
 """open_nodes package tests."""
 
 import unittest
+
 import mock
 
-from .. import node_class, all_nodes_types
-from ..node_m3 import NodeM3
-from ..node_a8 import NodeA8
+from gateway_code.nodes import open_node_class, all_open_nodes_types, OpenNode, all_control_nodes_types, \
```

Comment:
pep8: line is too long

You can use multi line with imports inside parenthesis:
```python
from gateway_code.nodes import (open_node_class, all_open_nodes_types
                                OpenNode, all_control_nodes_types, control_node_class)
```